src/main/java/ee/tuleva/onboarding/mandate/MandateService.java

```diff
@@ -203,7 +203,7 @@ private MandateSignatureStatus handleUnsignedMandateIdCard(
 
   private void notifyAboutSignedMandate(User user, Mandate mandate, Locale locale) {
     applicationEventPublisher.publishEvent(
-        new AfterMandateSignedEvent(this, user, mandate, locale));
+        new AfterMandateSignedEvent(this, user, mandate, locale, false));
```

Comment:
What is this `false`? Let's avoid using boolean parameters, they are hard to understand

Reply:
Instead of true/false you could use `batchId` as a parameter and add a method `isPartOfBatch()`

Reply:
Yup, agreed. Moved it to mandate and added a method there